src/server/services/tips/tips-service.test.ts

```diff
@@ -5,6 +5,16 @@ import WPI_E from '../../mock-data/json/wpi-e-aanvragen.json';
 import VERGUNNINGEN from '../../mock-data/json/vergunningen.json';
 import { createTipsFromServiceResults } from './tips-service';
 
+import {
+  vi,
+  describe,
+  beforeAll,
+  beforeEach,
+  it,
+  expect,
+  afterAll,
+} from 'vitest';

```

Comment:
We kunnen eventueel ook de configuratie van vitest aanpassen zodat dit niet expliciet nodig is in elk test bestand: https://vitest.dev/config/#globals

Reply:
Die staan al aan maar krijg het niet voor elkaar om VSCode de autoimport te laten herkennen / ts compiler de globals te laten herkennen